web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties

```diff
@@ -996,6 +982,8 @@ your_listed_crf_in_the_file = Your listed Event CRF in the file {0} does not exi
 
 your_message_sent_succesfully = Your message has been sent successfully.
 
+your_message_sent_succesfully_to = Your message has been sent successfully {0}.
+
```

Comment:
This string does not appear to be referenced in the code base.  Should be removed.